study.md

```diff
@@ -40,5 +43,6 @@ In your own words explain what a prototype is.  In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+A prototype is another invisible propery of every created object that is used to
+pass values and control inheritance in objects. 
```

Comment:
Prototype is not invisible it just gets created automatically. We can look at it like any other property though. When using constructor functions the function has a prototype object on it that stores the shared methods for every instance to inherit from.